Frameworks/include/CGContextInternal.h

```diff
@@ -57,6 +62,9 @@ class __CGContext : private objc_object {
     float scale;
     CGContextImpl* _backing;
 
+    // Reduces the number of BeginDraw() and EndDraw() calls needed, by counting in a stack-like manner
+    uint32_t _beginEndDrawDepth = 0;
```

Comment:
i'll std::atomic it

---
In reply to: [96093839](https://github.com/Microsoft/WinObjC/pull/1705#discussion_r96093839) [](ancestors = 96093839)